### DISCUSSION TITLE
Can I record in LOG mode with MLM2 / Max Lens Mod 2.0 ?

### DISCUSSION THREAD

**Discussion by TheNewAgeLight** [gopro/labs#743] [Q&A] [answered]
Been researching how to record in LOG mode with the Max Lens Mod enabled on the GoPro 12 but have been unsuccessful to finding the procedure. Does anybody know if this is possible?

Thank you & have a great day!

**Comment by dnewman-gpsw** [comment, marked as answer]
No & Yes.  GP-Log official mode is not available for MLM2.0 modes.  However with Labs installed, you can run the command
   ```cFoMLOGB=400oMWIDE=1```
which will enable the wide gamut log mode for all shooting modes on the camera.

**Reply by TheNewAgeLight** [reply]
Thank you so much! I will give this a go!

**Reply by TheNewAgeLight** [reply]
Heck yes that was surprisingly easy. Major upgrade. Thank you so much and nice job on the app for the QR codes!

**Reply by dimepues** [reply]
1. What is the color space supposed to show up as when processing the video? I have added that code via both the labs website and the ios app but still see rec 709 when I check the video on my computer.
2. I do see the 'features enabled: wide, log 400" when the camera boots
3. what should the camera U look like when it is configured correctly? If I only set the above and no other settings, I see the normal 4K / 10 bit. If I set the mode in the QR to log and / or the color depth to 10bit, the camera UI shows '4K (8bit)"
4. can someone share a qr code and share what their camera UI and video preview looks like with this on? 

thanks  😊

<img width="363" alt="Screenshot 2024-02-11 at 4 26 14 AM" src="https://github.com/gopro/labs/assets/5528680/27cf2a4e-a401-403d-8af7-9ba77f9d746b">

**Reply by dnewman-gpsw** [reply]
Wide Gamut does not change the camera metadata from Rec709, as the wide gamut is not a standard any playback tool would recognize. This is the same at most of the log curve wide gamut mode on most DLSRs. So completely typcial.

> If I set the mode in the QR to log and / or the color depth to 10bit, the camera UI shows '4K (8bit)

This sounds wrong.  cFoMLOGB=400oMWIDE=1 doesn't modify 8 or 10 bit settings, although you should only use LOGB with a 10-bit mode, you can applied to any mode.  No command in the oMwxyz=??? format will modify anything in the UI.

**Reply by TheNewAgeLight** [reply]
Hello again! Hope all is well!

Going to film another video after a firmware update today on my GoPro 12 with the Media Mod & Max Lens Mod 2.0 installed and enabled, getting the error "Invalid QR code" scanning my old saved code, and also scanning a new QR code generated with the confirmed "cFoMLOGB=400oMWIDE=1" command. 
Do you happen to know if there was any modification to the firmware or app that alters the command to enable LOG with Max Lens Mod 2.0 enabled?

Thank you & hope you have a great week ahead!

**Reply by dnewman-gpsw** [reply]
That is not an invalid code, you must be entering it wrong. 
![image](https://github.com/gopro/labs/assets/10856084/9d8f277a-3d4f-4ff5-ad0e-2f9f2f1ace3a)

**Reply by wrldh** [reply]
Is it possible to use it on gopro 13 + ultra wide lens too?

**Reply by dnewman-gpsw** [reply]
The Lens named **Ultra Wide** is for HERO13 only. https://www.amazon.com/GoPro-Ultra-Wide-HERO13-Black/dp/B0DCLL5DV4
The older **Max Lens Mod 2.0** also works on HERO13 https://www.amazon.com/GoPro-Max-Lens-HERO12-Black/dp/B0CFNFYMJD (but this lens lacks the auto camera configuration of the newer lens.)
The oldest original Max Lens Mod doesn't work in HERO13.

**Reply by wrldh** [reply]
Yes, I understand. What I was trying to ask is whether it’s possible to use LOG with the GoPro 13 and an Ultra Wide lens. Is there any command to activate what you called the "wide gamut log mode" in Labs for the GoPro 13?

**Reply by dnewman-gpsw** [reply]
Log and Ultra Wide on HERO13 is a standard feature. You can use Labs feature in the same way. All supported.
